content/en/blog/2024/cve-2024-36129/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Security vulnerability in confighttp (CVE-2024-36129)
-linkTitle: Security vulnerability in confighttp
+title: Security vulnerability in confighttp and configgrpc (CVE-2024-36129)
+linkTitle: Security vulnerability in confighttp and configgrpc
```

Comment:
While at it I think we coulda also mention that it's about Collector as it won't be clear just from the title otherwise.

Reply:
```suggestion
title: Security vulnerability in confighttp and configgrpc in the OpenTelemetry Collector (CVE-2024-36129)
linkTitle: Security vulnerability in confighttp and configgrpc in the OpenTelemetry Collector
```

Reply:
`linkTitle` should be short

Reply:
```suggestion
title: Security vulnerability in confighttp and configgrpc in the OpenTelemetry Collector (CVE-2024-36129)
linkTitle: Security vulnerability in HTTP and gRPC in the Collector
```
Maybe?

Reply:
I went with an alternative, I am fine with yours Armin. Up to Severin :)